<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>second_screen</class>
 <widget class="QWidget" name="second_screen">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>716</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="0,1,0,0">
   <property name="leftMargin">
    <number>9</number>
   </property>
   <property name="topMargin">
    <number>9</number>
   </property>
   <item>
    <widget class="QLabel" name="screen_label">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Please select devices to use</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="alldevice_layout">
     <property name="spacing">
      <number>6</number>
     </property>
     <property name="leftMargin">
      <number>40</number>
     </property>
     <property name="bottomMargin">
      <number>362</number>
     </property>
     <item>
      <layout class="QVBoxLayout" name="device_layout1">
       <item>
        <widget class="QCheckBox" name="deviceone_checkbox">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Maximum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Device 1</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="device_multichannel_layout1">
         <property name="leftMargin">
          <number>20</number>
         </property>
         <property name="rightMargin">
          <number>400</number>
         </property>
         <item>
          <widget class="QCheckBox" name="deviceone_multicheckbox">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
             <horstretch>1</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Use multiple channels?</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="deviceone_multiselect">
           <item>
            <property name="text">
             <string>1</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>2</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>3</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>4</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>5</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="device_layout2">
       <item>
        <widget class="QCheckBox" name="devicetwo_checkbox">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Maximum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Device 2</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="device_multichannel_layout2">
         <property name="leftMargin">
          <number>20</number>
         </property>
         <property name="rightMargin">
          <number>400</number>
         </property>
         <item>
          <widget class="QCheckBox" name="devicetwo_multicheckbox">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
             <horstretch>1</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Use multiple channels?</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="devicetwo_multiselect">
           <item>
            <property name="text">
             <string>1</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>2</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="device_layout3">
       <item>
        <widget class="QCheckBox" name="devicethree_checkbox">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Maximum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Device 3</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="device_multichannel_layout3">
         <property name="leftMargin">
          <number>20</number>
         </property>
         <property name="rightMargin">
          <number>400</number>
         </property>
         <item>
          <widget class="QCheckBox" name="devicethree_multicheckbox">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
             <horstretch>1</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Use multiple channels?</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="devicethree_multiselect">
           <item>
            <property name="text">
             <string>1</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>2</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>3</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>4</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="device_layout4">
       <item>
        <widget class="QCheckBox" name="device4_checkbox">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Maximum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Device 4</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="device_multichannel_layout4">
         <property name="leftMargin">
          <number>20</number>
         </property>
         <property name="rightMargin">
          <number>400</number>
         </property>
         <item>
          <widget class="QCheckBox" name="device4_multicheckbox">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
             <horstretch>1</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Use multiple channels</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="device4_multiselect">
           <item>
            <property name="text">
             <string>1</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>2</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="information_label">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Information Label</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="button_layout">
     <property name="leftMargin">
      <number>325</number>
     </property>
     <property name="rightMargin">
      <number>325</number>
     </property>
     <item>
      <widget class="QPushButton" name="next_button">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Next</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
